### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Daniil-Pankieiev**

### CODE REVIEW

**Review by prochigor — CHANGES_REQUESTED** [review]
in line 27 you have return but in annotation there is None, return is not nececery
annotation like that better to read:
def __init__(self,
             name: str,
             health: int = 100,
             hidden: bool = False
        ) -> None:

**Review by prochigor — APPROVED** [review]
Great

File: app/main.py [hunk 1]
```diff
@@ -1 +1,33 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+            self, name: str,
+            health:
+            int = 100,
```
[the remark this answers is not in the document]
Comment (Daniil-Pankieiev) [reply app/main.py]: Fixed

**Review by fsocie7y — APPROVED** [review]
well done ^_^

**Pull request closed without merging**